mkt/api/authorization.py

```diff
@@ -177,6 +165,40 @@ def __call__(self, *a):
         return self
 
 
+class GroupPermission(GroupPermissionBase):
+
+    def __init__(self, app, action):
+        self.app = app
+        self.action = action
+
+
+class AllowReviewerReadOnly(GroupPermissionBase):
+    app = 'Apps'
+    action = 'Review'
+
+    def has_permission(self, request, view):
+        return (request.method in SAFE_METHODS and
+                super(AllowReviewerReadOnly, self).has_permission(request,
+                                                                  view))
+
+
+class AllowStaffReadOnlyIfReviewed(GroupPermissionBase):
+    """
+    Allow users in the Staff group to view apps that have any published status.
+    """
+    app = 'Apps'
+    action = '%'
```

Comment:
iirc `"Apps:%"`  is shorthand for *any* `Apps:...` permission, of which many of the holders won't be staff (reviewers, moderators, etc). 'Apps:Edit' is probably the closest 'staff' permission.